parser.cpp

```diff
@@ -159,9 +159,13 @@ void Stripper::SetEntityList(const char *ents)
     char *_tmp = NULL;
     size_t _tmpsize = 0;
     size_t pos = 0;
+    char in_string = 0;
```

Comment:
I guess it'd make more sense, yeah. I used char because bool is not a native C type. (https://stackoverflow.com/questions/1608318/is-bool-a-native-c-type)

Though I guess doing in_string ^= true; would look kinda weird whereas with 0 and 1 it makes more sense :p